releases/sui/2025-11-Sui-Role-Transfers-Release-v1.0.0.md

```diff
@@ -0,0 +1,263 @@
+| **Mainnet**          | `0x980372415053fe9d09956dea38d33d295f10de3d5c5226099304fe346ce241c9` | TBD                                                                  |
+
+```bash
+GS_UPG_CAP_ID=$(jq -r '.chains["sui"].contracts.GasService.upgradeCapId' ./axelar-chains-config/info/$ENV.json)
+
+# Using helper script by contract/object names
+ts-node sui/transfer-object.js --contractName GasService --objectName UpgradeCap --recipient "$TARGET_ADDRESS"
+
+# Or using direct object id
+ts-node sui/transfer-object.js --objectId "$GS_UPG_CAP_ID" --recipient "$TARGET_ADDRESS"
+
+sui client object "$GS_UPG_CAP_ID"
+```
+
+### Step 5(Optional): Transfer Operators OwnerCap to Relayer Operators EOA 
+
+**New Owner**: Relayer Operators EOA
+
+| Network              | Current Owner                                                        | Target Address |
+| -------------------- | -------------------------------------------------------------------- | -------------- |
+| **Devnet Amplifier** | `0x619592640cab068848c92c309bdd665d6a5e3f2f2f51ec9464cc112166daf7d1` | TBD            |
+| **Stagenet**         | `0x619592640cab068848c92c309bdd665d6a5e3f2f2f51ec9464cc112166daf7d1` | TBD            |
+| **Testnet**          | `0x619592640cab068848c92c309bdd665d6a5e3f2f2f51ec9464cc112166daf7d1` | TBD            |
+| **Mainnet**          | `0xd7b392db51562a72e50f310e78c827b4e917254cf15c5cec6c97964299a6be2a` | TBD            |
+
+```bash
+OPERATORS_OWNERCAP_ID=$(jq -r '.chains["sui"].contracts.Operators.ownerCapId' ./axelar-chains-config/info/$ENV.json)
+RELAYER_OPERATORS_EOA=<TARGET_ADDRESS>
+
+# Using helper script by contract/object names
+ts-node sui/transfer-object.js --contractName Operators --objectName OwnerCap --recipient "$RELAYER_OPERATORS_EOA"
+
+# Or using direct object id
+ts-node sui/transfer-object.js --objectId "$OPERATORS_OWNERCAP_ID" --recipient "$RELAYER_OPERATORS_EOA"
+
+sui client object "$OPERATORS_OWNERCAP_ID"
+```
+
+### Step 6(Optional): Transfer Operators UpgradeCap to Relayer Operators EOA
+
+**New Owner**: Relayer Operators EOA
+
+| Network              | Current Owner                                                        | Target Address |
+| -------------------- | -------------------------------------------------------------------- | -------------- |
+| **Devnet Amplifier** | `0x1471a8acf730a05a7d720e52c7ef94024c7351502c83b80da5583db2f6b0b8df` | TBD            |
+| **Stagenet**         | Not set in config                                                    | TBD            |
+| **Testnet**          | Not set in config                                                    | TBD            |
+| **Mainnet**          | Not set in config                                                    | TBD            |
+
+```bash
+OPERATORS_UPG_CAP_ID=$(jq -r '.chains["sui"].contracts.Operators.upgradeCapId' ./axelar-chains-config/info/$ENV.json)
```

Comment:
We can remove this transfer step. Operators contract is not version controlled (non-upgradable), so there is no `UpgradeCap` for it ([see objects](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/axelar-chains-config/info/mainnet.json#L2680-L2683))